# Generated by Django 2.0.3 on 2018-06-01 20:58

from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='Payment',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('amount', models.IntegerField(editable=False, verbose_name='Сумма в копейках')),
                ('order_id', models.CharField(editable=False, max_length=100, unique=True, verbose_name='Номер заказа')),
                ('description', models.TextField(blank=True, default='', editable=False, max_length=250, verbose_name='Описание')),
                ('success', models.BooleanField(default=False, editable=False, verbose_name='Успешно проведен')),
                ('status', models.CharField(default='', editable=False, max_length=20, verbose_name='Статус транзакции')),
                ('payment_id', models.CharField(default='', editable=False, max_length=20, verbose_name='Уникальный идентификатор транзакции в системе Банка')),
                ('error_code', models.CharField(default='', editable=False, max_length=20, verbose_name='Код ошибки')),
                ('payment_url', models.CharField(blank=True, default='', editable=False, help_text='Ссылка на страницу оплаты. По умолчанию ссылка доступна в течении 24 часов.', max_length=100, verbose_name='Ссылка на страницу оплаты.')),
                ('message', models.TextField(blank=True, default='', editable=False, verbose_name='Краткое описание ошибки')),
                ('details', models.TextField(blank=True, default='', editable=False, verbose_name='Подробное описание ошибки')),
            ],
            options={
                'verbose_name_plural': 'Платежи',
                'verbose_name': 'Платеж',
            },
        ),
    ]
